src/brevitas/proxy/groupwise_float_parameter_quant.py

```diff
@@ -28,6 +28,7 @@ def apply_input_view(self, x):
         return x.flatten(start_dim, start_dim + 1)
 
     def create_quant_tensor(self, qt_args: Tuple[Any]) -> GroupwiseFloatQuantTensor:
+        shape = self.tracked_parameter_list[0].shape
```

Comment:
We don't support weight quant sharing for groupwise anyway, so this is safe, but it is ugly.